contracts/samples/utils/OracleHelper.sol

```diff
@@ -100,11 +100,10 @@ abstract contract OracleHelper {
             oracleHelperConfig.tokenOracleReverse,
             oracleHelperConfig.nativeOracleReverse
         );
-        uint256 priceNewByOld = price * PRICE_DENOMINATOR / _cachedPrice;
-
+        uint256 priceRatioNewByOld = price * PRICE_DENOMINATOR / _cachedPrice;
         bool updateRequired = force ||
-            priceNewByOld > PRICE_DENOMINATOR + priceUpdateThreshold ||
-            priceNewByOld < PRICE_DENOMINATOR - priceUpdateThreshold;
+            priceRatioNewByOld > PRICE_DENOMINATOR * (1 + priceUpdateThreshold / 1e3) ||
```

Comment:
I think you will just get `priceUpdateThreshold / 1e3 == 0` for any `priceUpdateThreshold < 1e3`, am I missing something?

Reply:
Indeed. There was an issue before as well. I think I fixed it.